common/v2/services/Store/Asset/helpers.ts

```diff
@@ -61,9 +61,9 @@ export const getAssetByContractAndNetwork = (
 export const getTotalByAsset = (assets: StoreAsset[]) =>
   assets.reduce(
     (dict, asset) => {
-      const prev = dict[asset.uuid];
+      const prev = dict[asset.name];
```

Comment:
Since it's the `uuid` that is unique, I don't understand why we would use the `name`.
The previous implementation worked for wallet breakdown, and currently it's the only place this function is implemented.